### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by maxakuru**
- fixes max-width issues in header, footer, breadcrumbs
- fixes bug where breadcrumbs didn't render in PDP
- update PDP to show all data in staging

related api pr: https://github.com/hlxsites/eecol-cloudflare-worker/pull/32

Test URLs:
- Before: https://main--eecol--hlxsites.hlx.page/ca/en
- After: https://bug-fixes--eecol--hlxsites.hlx.page/ca/en

### CODE REVIEW

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
-        </span>
-      </a>
-    </p>
-    <p>
-      <a href="https://ca.linkedin.com/company/eecol-electric">
-        <img src="/icons/social-linkedin.svg" width="24" height="24">
-      </a>
-    </p>
-  `;
+  const container = html`
```
Comment (mbouxsein24) [inline blocks/footer/footer.js:5]: Future social component?

File: blocks/header/header.css [hunk 2]
```diff
@@ -58,7 +58,7 @@ header nav[aria-expanded='true'] {
 
 /* topbar */
 header .nav-topbar {
-  max-width: 1640px;
+  max-width: 1344px;
```
Comment (mbouxsein24) [inline blocks/header/header.css:61]: We should be using more variables.  I asked Johnny if he wanted to use our existing but he was leaning towards starting over.

**Review by mbouxsein24 — APPROVED** [review]

File: blocks/footer/footer.js (hunk 1 above)
Comment (maxakuru) [reply blocks/footer/footer.js:5]: yeah -- I think we want to localize the footer doc and include the socials in there.. the way it is now won't work with other languages

File: blocks/header/header.css (hunk 2 above)
Comment (maxakuru) [reply blocks/header/header.css:61]: existing from aem? or existing in this project?
definitely agree, I'll actually make that one now since I'm in there

**Pull request merged**